pyntc/devices/f5_device.py

```diff
@@ -299,12 +301,12 @@ def _upload_image(self, image_filepath):
                 headers["Content-Range"] = content_range
                 # pylint: disable=missing-timeout
                 # TODO Add timeout to requests.post, missing timeout can cause the method to hang indefinitely
-                requests.post(
+                requests.post(  # noqa: S113
```

Comment:
Maybe we should finally just add a sane timeout instead of adding another ignore?

Reply:
yea...maybe in the next PR. Was trying to avoid any code changes in this PR for cleaniness.